tests/kernel/smp/src/main.c

```diff
@@ -1004,8 +1014,12 @@ static void inc_global_cnt(void *a, void *b, void *c)
 	}
 }
 
-static int run_concurrency(int type, void *func)
+static int run_concurrency(void *p1, void *p2, void *p3)
 {
+	ARG_UNUSED(p3);
+
+	int type = p1;
+	void *func = p2;
```

Comment:
`k_thread_entry_t func = p2;` for the sake of clarity ?

Reply:
done